pkg/pillar/zedcloud/send.go

```diff
@@ -82,6 +82,17 @@ func SendOnAllIntf(ctx ZedCloudContext, url string, reqlen int64, b *bytes.Buffe
 			}
 		}
 		for _, intf := range intfs {
+			portStatus := ctx.DeviceNetworkStatus.GetPortByIfName(intf)
+			if portStatus != nil {
+				if portStatus.CloudUnReachable == true {
+					log.Infof("SendOnAllIntf: XXXXX Skipping interface %s, since cloud is not "+
```

Comment:
Do we need this as log.Infof? We already know it is now reachable. May be just log.Debugf?